app/views/documents/_index.html.erb

```diff
@@ -16,20 +17,64 @@
     <div class="row-wrapper">
       <div class="tab-content" id="myTabContent">
         <% Document.category.values.each_with_index do |category, index| %>
-          <div class="tab-pane fade show <%= 'active' if index.zero? %>" id=<%= category %> role="tabpanel" aria-labelledby="<%= category %>-tab">
+          <div class="tab-pane fade show <%= 'active' if index.zero? %>"
+               id=<%= category %> 
+               role="tabpanel" 
+               aria-labelledby="<%= category %>-tab">
             <div class="row cols-md-space cols-sm-space cols-xs-space">
               <table id="documents" class="table table-sm table-hover">
                 <thead>
-                  <tr>
-                    <th scope="col">Nom du document</th>
-                    <th scope="col">Télécharger</th>
+                  <tr >
+                    <th scope="col">
+                      <li style="list-style: none;">
+                        <a class="sort-link name" 
+                           data-category="<%= category %>"
+                           href="#"
+                           data-remote="true"
```

Comment:
`data-remote: true` should not be needed here, as you are not making any backend request